database/csv-generator/faker-schema.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const faker = require('faker');
+
+const product = (imgCount = 0, totImg = 0) => {
+  let randImg = randImgList(imgCount, totImg);
+  return {
+    name: faker.commerce.productName(),
+    detail: faker.lorem.slug(),
+    image: randImg,
+  };
+};
+
+const randImgList = (imgNum, totImg) => {
+  let result = new Set();
+  while(result.size < imgNum) {
+    result.add(Math.floor(Math.random()*totImg));
```

Comment:
Can you use the shuffle deck algorithm to pick random images? It may be more efficient.

Reply:
How would i use shuffle deck to pick random images, not sure actually.

Reply:
I would refer to the solution to the toy problem. What it does is eliminate repeats, so you would not even need to use a set. However, your solution works, so it's low priority.